data/registry/instrumentation-dotnet-nservicebus.yml

```diff
@@ -0,0 +1,24 @@
+# cSpell:ignore # add words unknown to cspell, remove this line if not required
+title: NServiceBus OpenTelemetry Integration # the name of your project
+registryType: <exporter/core/instrumentation> # the type of integration; is this an exporter, plugin, API package, or something else?
```

Comment:
@svrnm what's your take on how this should be tagged?

Reply:
```suggestion
registryType: instrumentation # the type of integration; is this an exporter, plugin, API package, or something else?
```